### ISSUE TITLE
Non-debug logging

### ISSUE THREAD

**Issue by JonLaliberte** [bitpay/magento-plugin#115]
In `/app/code/community/Bitpay/Core/Model/Observer.php`

There are 2 log statements that should not be there.

```
Mage::log('sales_order_place_after--------------:');
Mage::log('$paymentCode:' . $paymentCode);
```

**Comment by pieterpoorthuis** [comment]
Also please include the following in the debug log:
* PHP version
* Magento version
* BitPay plugin version
* Livenet / testnet configuration in sendPairingRequest

**Comment by HeliumRaccoon** [comment]
this code is buggy you should remove bugs

**Comment by pieterpoorthuis** [comment]
Fixed in release 2.1.18
